net/init-metrics.sh

```diff
@@ -83,11 +83,16 @@ else
   query "CREATE DATABASE \"$netBasename\""
   query "ALTER RETENTION POLICY autogen ON \"$netBasename\" DURATION 7d"
   query "GRANT READ ON \"$netBasename\" TO \"ro\""
-  query "GRANT WRITE ON \"$netBasename\" TO \"scratch_writer\""
+  query "GRANT WRITE ON \"$netBasename\" TO \"${username}\""
 
-  SOLANA_METRICS_CONFIG="host=$host,db=$netBasename,u=scratch_writer,p=topsecret"
+  SOLANA_METRICS_CONFIG="host=$host,db=$netBasename,u=${username},p=${password}"
+  set +x
+  echo "Consider adding \`SOLANA_METRICS_CONFIG\` to your shell environment for future use:"
+  echo "export SOLANA_METRICS_CONFIG=\"$SOLANA_METRICS_CONFIG\""
 fi
 
+set -x
+sed -i '/SOLANA_METRICS_CONFIG/d' "${configFile}"
```

Comment:
*minor*

I would be a bit more precise with the file editing.
As written, it would remove any line that contains `SOLANA_METRICS_CONFIG`.
As the intent is to update or append a specific line, it not match the intent exactly.
I think adding a few more details to the line selection could avoid a potentially confusing bug when the wrong line is accidentally deleted.

```suggestion
sed -i '/^export\s\+SOLANA_METRICS_CONFIG\s*=\s*"/d' "${configFile}"
```

Reply:
Thanks for the fix, that was indeed a good catch!